src/Glpi/Form/Destination/AbstractCommonITILFormDestination.php

```diff
@@ -329,7 +329,7 @@ private function applyPredefinedTemplateFields(array $input): array
         /** @var \CommonITILObject $itil */
         $itil = new $itemtype();
         $template = $itil->getITILTemplateToUse(
-            entities_id: $_SESSION["glpiactive_entity"],
+            entities_id: $_SESSION["glpiactive_entity"] ?? 0,
```

Comment:
The template should be fetched from the target entity (it is configurable in forms).